container/sagemaker/python_service.py

```diff
@@ -155,7 +159,25 @@ def on_post(self, req, res):
             res.status = falcon.HTTP_507
             res.body = json.dumps({
                 'error': str(e)
-            }).encode('utf-8')
+            }).encode(encodings.utf_8.getregentry().name)  # pylint: disable=E1101
+
+    def _read_model_config(self):
+        models = []
+        name_key = re.compile(r'( *)name:(.*)')
```

Comment:
Awesome! I think this is much cleaner =)

[Minor] If we think the class can have tabs, we should do
```
re.compile(r'([ \t]*)name:(.*)')
```